.github/markdown-links.config.json

```diff
@@ -2,6 +2,9 @@
   "ignorePatterns": [
     {
       "pattern": "^https?:\/\/.*"
+    },
+    {
+      "pattern": "^https?://(127.0.0.1|localhost)[:/].*"
```

Comment:
No need to add the pattern here. As you can see, all urls that start with `http` are ignored already.